modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -223,7 +223,7 @@ The value to select for this provider is: `ocLdapPermissionManager`.
 
 The SMB protocol supports registering for notifications of file changes on remote Windows SMB storage servers. Notifications are more efficient than polling for changes, as polling requires scanning the whole mounted SMB storage. While files changed through the ownCloud Web Interface or sync clients are automatically recognized by ownCloud, recognition is not possible when files are changed directly on remote SMB storage mounts. When using the _listener_, files changed on the SMB backend are recognized and a notification is stored in the database. The _process-queue_ job reads these stored notifications and initiates further actions.
 
-NOTE: The capability of the listener depends on the ability of the used SMB/CIFS storage backend to provide notifications. While Windows file servers have no limitations, some vendors may have restrictions. Please check these with your storage provider. It may be possible, that notifications for Samba only work for the target folder you're listening to, but not for any sub structures. If you're listening on the "/top" folder, you may not receive notifications for "/top/middle/bottom" folder. In this case, you have to setup listeners for every _existing_ folder and also for any _new_ folders that will be created. With Windows file servers, you will receive notifications for every file or subfolder inside the folder you're listening to.
+NOTE: The capability of the listener depends on the ability of the used SMB/CIFS storage backend to provide notifications. While Windows file servers have no limitations, some vendors may have restrictions. Please check these with your storage provider. It may be possible, that for example notifications for Samba only work for the target folder you're listening to, but not for any sub structures. If you're listening on the "/top" folder, you may not receive notifications for "/top/middle/bottom" folder. In this case, you have to setup listeners for every _existing_ folder and also for any _new_ folders that will be created. With Windows file servers, you will receive notifications for every file or subfolder inside the folder you're listening to.
```

Comment:
```suggestion
NOTE: The capability of the listener depends on the ability of the used SMB/CIFS storage backend to provide notifications. While Windows file servers have no limitations, some vendors may have restrictions. Please check these with your storage provider. It may be possible, that for example notifications for Samba only work for the target folder you're listening to, but not for any sub structures. If you're listening on the "/top" folder, you may not receive notifications for "/top/middle/bottom" folder. In this case, you have to set up listeners for every _existing_ folder and also for any _new_ folders that will be created. With Windows file servers, you will receive notifications for every file or subfolder inside the folder you're listening to.
```